generator-cas-app/generators/app/templates/.tool-versions

```diff
@@ -0,0 +1,4 @@
+nodejs <%= nodejs %>
```

Comment:
should all of our template files have the `.ejs` extension?

Reply:
No it doesn't need to.